next/pages/api/ginis/official-board-file/[fileId]/[fileName].ts

```diff
@@ -0,0 +1,49 @@
+import { getOfficialBoardFileBase64Encoded } from '@backend/ginis/server/getOfficialBoardFileBase64Encoded'
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+/**
+ * This handler serves files from official board (Ginis UDE = Úřední deska = Úradná tabuľa) as pdf.
+ * The api url is publicly visible, and therefore it is in slovak language.
```

Comment:
likely a forgotten comment, or a file rename is required (the api url is in english at the moment)